scripts/notify-ci-wave.mjs

```
@@ -159,13 +181,148 @@ function parseReleaseContributors(value) {
+      key.endsWith('[bot]') ||
+      NON_HUMAN_GITHUB_LOGINS.has(key)
+    )
+      continue;
     seen.add(key);
     contributors.push(login);
   }
   return contributors;
 }
 
+async function githubApi(repository, path) {
+  const response = await fetch(
```

ESLint

[error] 'fetch' is not defined.

(no-undef)